packages/x-data-grid/src/components/columnsManagement/GridColumnsManagement.tsx

```diff
@@ -207,27 +210,64 @@ function GridColumnsManagement(props: GridColumnsManagementProps) {
     }
     return false;
   };
+  const handleSearchReset = React.useCallback(() => {
+    setSearchValue('');
+    searchInputRef.current!.focus();
+  }, []);
 
   return (
     <React.Fragment>
       <GridColumnsManagementHeader className={classes.header} ownerState={rootProps}>
         <rootProps.slots.baseTextField
           placeholder={apiRef.current.getLocaleText('columnsManagementSearchTitle')}
+          aria-label={apiRef.current.getLocaleText('columnsManagementSearchTitle')}
```

Comment:
Thanks for adding a label. I think we want to apply this to the `input` element itself. Perhaps we can add it as a property of `InputProps` below?

Reply:
I have applied it in `inputProps`, by applying to `InputProps` label still added to the div wrapper.

Reply:
Good spot. Thanks!